### PR TITLE
fix(nx-plugin-openapi): fix silent option generation

### PR DESCRIPTION
**Pull request by youngchen7**

### CODE REVIEW

File: packages/nx-plugin-openapi/src/test/mockSpawn.ts [hunk 1]
```diff
@@ -6,21 +6,23 @@ export function mockSpawn(
...
+    mock.mockImplementationOnce((command: string, args: string[], options: { stdio: 'ignore' | 'pipe' }) => {
       expect([command, ...args]).toEqual([invocation.command, ...invocation.args]);
 
       const child: any = new EventEmitter();
       child.stdin = new EventEmitter();
       child.stdin.write = jest.fn();
       child.stdin.end = jest.fn();
-      child.stdout = new EventEmitter();
-      child.stderr = new EventEmitter();
+      if (options.stdio !== 'ignore') {
```
Comment (youngchen7) [inline packages/nx-plugin-openapi/src/test/mockSpawn.ts:16]: This is becoming uncomfortably close to mocking the node spawn implementation details in my opinion

Comment (trumbitta) [reply packages/nx-plugin-openapi/src/test/mockSpawn.ts:16]: Right? At the time it seemed the only way for testing the executor. Do you have anything in mind?

**Comment by trumbitta** [comment]
@all-contibutors, I think it's time to also add @youngchen7 for bug fixing 🥇

**Comment by trumbitta** [comment]
@all-contributors, I think it's time to also add @youngchen7 for bug fixing 🥇

**Comment by trumbitta** [comment]
@all-contributors please add @youngchen7 for bug fixing

**Pull request merged**